lib/rubocop/cop/lint/to_json.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks to make sure `#to_json` includes an optional hash
+      # argument.  When overriding `#to_json`, callers may invoke JSON
+      # generation via `JSON.generate(your_obj)`.  Since `JSON#generate` allows
+      # for optional arguments, your method should too.
+      #
+      # @example
+      #   # bad
+      #   def to_json
+      #   end
+      #
+      #   # good
+      #   def to_json(opts = {})
+      #   end
+      #
+      class ToJSON < Cop
+        MSG = <<-M_JSON.strip_indent.chomp.freeze
```

Comment:
Let's change the identifier from `M_JSON` (makes me think of the late king of pop 😅) to `MESSAGE`. 🙂

Reply:
No need for a heredoc here:
```rb
MSG = '`#to_json` requires an optional hash argument to be parsable ' \
  'via JSON.generate(obj).'

```